px-data-grid.html

```diff
@@ -208,37 +208,75 @@
 
         static get properties() {
           return {
-
             /**
              * Data for the table to display.
              *
-             * Expected data format is a JSON array of objects. Each object in the array represents a row in the table.
+             * Pass an array of objects. Each object will be rendered as a row
```

Comment:
I'm not going to go through this file again - can you just run through the same list of suggestions from the other file for all the shared properties?